renovate.json

```diff
@@ -32,6 +32,7 @@
        {
```

Comment:
Maybe drop the rule instead of setting `enabled` to `false`?

Reply:
Let's keep it for now. We can easily enable it later when they issue with incorrect PR generated by renovate.